pkg/imageengine/buildah/sealer_image_extension.go

```diff
@@ -1,65 +0,0 @@
-// Copyright © 2022 Alibaba Group Holding Ltd.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package buildah
```

Comment:
What are the considerations behind this file modification？

Reply:
this file contains image extension-related information used at  `sealer inspect`, so i move those functions to inspect.go